baseline/pytorch/tagger/model.py

```diff
@@ -136,83 +129,358 @@ def make_input(self, batch_dict, perm=False, numpy_to_tensor=False):
             return example_dict, perm_idx
         return example_dict
 
-    def forward(self, input: Dict[str, torch.Tensor]) -> torch.Tensor:
-        return self.layers(input)
-
-    def compute_loss(self, inputs):
-        tags = inputs['y']
-        lengths = inputs['lengths']
-        unaries = self.layers.transduce(inputs)
-        return self.layers.neg_log_loss(unaries, tags, lengths)
-
     def get_vocab(self, vocab_type='word'):
+        """Produce a dictionary for the vocab feature name
```

Comment:
Does anything use this `get_vocab` method anymore? I though all the vocab was coordinated via the services?

Reply:
I think this can go away.  Will double check